src/app/core/services/session.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { SESSIONS_DATA } from '../../../assets/data/sessions';
+import { ISession } from '../../models/sessions';
+import { map } from 'rxjs/operators';
+
+@Injectable({ providedIn: 'root' })
+export class SessionService {
+
+  getSessions(): Observable<Array<ISession>> {
+    return of(SESSIONS_DATA);
+  }
+
+  getSessionById(id: number, key: string): Observable<Array<ISession>> {
```

Comment:
Probably this function should return Observable<ISession | indefined> rather than Observable<Array<ISession>>

Reply:
My fault, bad naming. According to my idea, it must return Observable<ISession[]> by cinemaId or movieId